open-api/rest-catalog-open-api.yaml

```diff
@@ -1482,6 +1490,33 @@ components:
       explode: false
       example: "vended-credentials,remote-signing"
 
+    page-token:
+      name: pageToken
+      in: query
+      description:
+        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables) by signaling to the service
+        that they would prefer response to be paginated.
+        
+        Clients will initiate the request by sending an empty pageToken e.g. GET /tables?pageToken or /tables?pageToken=
+        For servers that support pagination, they will recognize pageToken and honor the contracts specified above 
```

Comment:
wdyt about the following wording? 
`For servers supporting pagination, a NextPageToken will be returned if there are more results available. For servers not supporting pagination, the pageToken is ignored and all results are returned.`

Reply:
Thanks for the wording will try to fix it like this (including the details on  next request)

```
        For servers that support pagination, they will recognize pageToken and return a NextPageToken in response if there are more results available.
        After initial request, it is expected that the NextPageToken from the last response be used as the pageToken in the subsequent request.
        For servers that do not support pagination, they will ignore the pageToken and return all results.

```